src/Docker/Runner/Limits.php

```diff
@@ -91,11 +91,6 @@ public function getCpuLimit(Image $image)
         return min($instance, $project);
     }
 
-    public function getDeviceIOLimits(Image $image)
-    {
-        return '50m';
-    }
-
     /**
```

Comment:
vidis, taky to mohly byt 2 PR :)

Reply:
obojí ale řeší stejnou věc, tak jsem to nechtěl zbytečně salámovat